src/View/nav.html.twig

```diff
@@ -0,0 +1,52 @@
+<nav role='navigation'>
+	<div class="main">
+		<a href="" {# Enter link here #}>
+			<img src="../assets/images/logook.png" class="logo" alt="logo"></a>
+		<div class="search-box" id="search">
+			<img src="../assets/images/magglass.png">
+			<input type="text" class="input-search" placeholder="Type to Search...">
+			<button class="submit-search">Submit</button>
+		</div>
+		<div class="menu" id="icons" onclick="searchHide();">
+			<a href="" {# Enter link here #}><img src="../assets/images/Vector.png"></a>
```

Comment:
Could your wrap all thos A into a ul (descendant of an Li) and then wrap all a into li tags ?